src/debugger/titaniumDebugAdapter.ts

```diff
@@ -65,12 +65,12 @@
 		}
 	}
 
-	public async disconnect (args: DebugProtocol.DisconnectArguments): Promise<void> {
+	public override async disconnect (args: DebugProtocol.DisconnectArguments): Promise<void> {
 		await this.cleanup();
```

Comment:
<!--
  0 failure: 
  1 warning:  src/debugger/tita...
  
  
  DangerID: danger-id-Danger;
  File: src/debugger/titaniumDebugAdapter.ts;
  Line: 5;
-->

- :warning: src/debugger/titaniumDebugAdapter.ts line 5 – Unexpected use of 'require("url").URL'. Use the global variable 'URL' instead. (node/prefer-global/url)